cognite/client/utils/_experimental.py

```diff
@@ -10,21 +10,24 @@ def __init__(
         api_maturity: Literal["alpha", "beta", "General Availability"],
         sdk_maturity: Literal["alpha", "beta"],
         feature_name: str,
+        pluralize: bool = False,
```

Comment:
Have feature_name be a list of strings instead? Then you can do `name = ", ".join(feature_names)` and `verb = "are" if len(feature_names) > 0 else "is"`